x-pack/plugin/blob-cache/src/test/java/org/elasticsearch/blobcache/shared/SharedBytesTests.java

```diff
@@ -31,7 +31,8 @@ public void testReleasesFileCorrectly() throws Exception {
                 randomIntBetween(1, 16) * 4096L,
                 nodeEnv,
                 ignored -> {},
-                ignored -> {}
+                ignored -> {},
+                randomBoolean()
```

Comment:
Didn't put a guard against this on Windows boxes here, I think it's fine to let the maps live on until the test JVM ends with how few tests we run and how small the maps are in those.